apps/calendar/js/views/settings.js

```diff
@@ -27,6 +26,12 @@ Settings.prototype = {
 
   calendarList: null,
 
+  /**
+   * used internally to check if calendar and account data was loaded from db
+   * @private
+   */
+  _load: null,
```

Comment:
Maybe you should specify that this is a `Promise` instead of a `boolean` or some such?